src/main/java/io/fabric8/jenkins/openshiftsync/BuildSyncRunListener.java

```diff
@@ -201,9 +206,15 @@ private void upsertBuild(Run run, RunExt wfRunExt) {
       return;
     }
 
+    BluePipeline blueResource = (BluePipeline) BluePipelineFactory.resolve(run.getParent());
     String rootUrl = OpenShiftUtils.getJenkinsURL(getOpenShiftClient(), cause.getNamespace());
     String buildUrl = joinPaths(rootUrl, run.getUrl());
     String logsUrl = joinPaths(buildUrl, "/consoleText");
+    String logsRawUrl = joinPaths(buildUrl, "/console");
```

Comment:
(the problem we would seem to have is that we've been using the raw url and not labeling it as such.  so we need to introduce `/console` as being logsConsole or logsPretty or some such thing)